echo-pipelinetriggers/src/test/groovy/com/netflix/spinnaker/echo/pipelinetriggers/PipelineCacheSpec.groovy

```diff
@@ -114,5 +114,50 @@ class PipelineCacheSpec extends Specification implements RetrofitStubs {
     then:
     notThrown(JsonMappingException)
   }
+
```

Comment:
maybe you should add a test for the happy path?

Reply:
I guess it's kind of in this first one.

Reply:
yes that's how I think of the first test